disperser/common/blobstore/blob_metadata_store.go

```diff
@@ -59,11 +61,16 @@ func (s *BlobMetadataStore) QueueNewBlobMetadata(ctx context.Context, blobMetada
 	if s.shadowTableName != "" && s.shadowTableName != s.tableName {
 		err = s.dynamoDBClient.PutItem(ctx, s.shadowTableName, item)
 		if err != nil {
+			// TODO: should we return an error here? If not, then we should document why we are ignoring the error
```

Comment:
@ian-shim

Reply:
shadow tables aren't used. I will open a PR to clean them up

Reply:
https://github.com/Layr-Labs/eigenda/pull/830

Reply:
Thanks @ian-shim ! Removed the TODO here since this piece of code will be removed once we merge your PR. [d5a75f1](https://github.com/Layr-Labs/eigenda/pull/823/commits/d5a75f1af353d73b6684dad71d603e14e491ed08)